src/api/CMakeLists.txt

```diff
@@ -18,15 +18,20 @@ $<TARGET_OBJECTS:rewriters>
 
 add_library(api OBJECT "")
 
+if(PARALLEL)
+    target_link_libraries(api PUBLIC PTPLib::PTPLib)
+    target_compile_definitions(api PUBLIC -DCUBE_AND_CONQUER)
+endif()
+
 set(PRIVATE_SOURCES_TO_ADD
     "${CMAKE_CURRENT_SOURCE_DIR}/MainSolver.cc"
-    "${CMAKE_CURRENT_SOURCE_DIR}/MainSplitter.cc"
+    "${CMAKE_CURRENT_SOURCE_DIR}/MainSplitter.hpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/PartitionManager.cc"
 )
 
 set(PUBLIC_SOURCES_TO_ADD
     "${CMAKE_CURRENT_SOURCE_DIR}/MainSolver.h"
-    "${CMAKE_CURRENT_SOURCE_DIR}/MainSplitter.h"
+    "${CMAKE_CURRENT_SOURCE_DIR}/MainSplitter.hpp"
```

Comment:
Can we just call the file `MainSplitter.h` here?  The `.hpp` seems to be inconsistent with the rest of opensmt codebase.

Reply:
Sure